python/amici/ode_export.py

```diff
@@ -3398,6 +3398,6 @@ def _custom_pow_eval_derivative(self, s):
 
     return part1 + sp.Piecewise(
         (
-            sp.sympify(0.0), sp.And(sp.Eq(self.base, 0), sp.Eq(dbase, 0))),
+            sp.Float(0.0), sp.And(sp.Eq(self.base, 0), sp.Eq(dbase, 0))),
```

Comment:
Just nitpicking :wink: but since we are still at the symbolic stage why do you think float is better here?

Reply:
I generally dislike integers, bad habit from dividing them too often in C.